manifests/bind_event.pp

```diff
@@ -6,30 +6,26 @@
 #
 # $queue::                      Name of the event queue
 #
+# $exchange::                   Name of the exchange the queue is on
+#
 # $hostname::                   Set to localhost for qpid-config operations
 #
 # $port::                       Port that qpid is listening on
 #
 # $ssl_cert::                   SSL cert to use for qpid-config commands
 define qpid::bind_event(
   $queue,
+  $exchange = 'event',
   $hostname = 'localhost',
-  $port = 5671,
+  $port = undef,
```

Comment:
Because the port is determined in the wrapper. That means it automatically selects the correct port for SSL and non-SSL.